tests/python/gaia-ui-tests/gaiatest/mocks/mock_call.py

```diff
@@ -4,20 +4,19 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import time
-from datetime import date
+import time, datetime
 
 
 class MockCall(dict):
 
-    def __init__(self, phone_number=None, call_type='incoming', date=date.today(), duration=1,
+    def __init__(self, phone_number=None, call_type='incoming', date=datetime.date.today(), duration=1,
                  status='cancelled', serviceId=0, emergency=False, voicemail=False, **kwargs):
         super(MockCall, self).__init__(self, **kwargs)
 
         current_time = repr(time.time()).replace('.', '')
         self['number'] = phone_number if phone_number is not None else '555%s' % current_time[-7:]
         self['type'] = call_type
-        self['date'] = '{:%m/%d/%Y}'.format(date)
+        self['date'] = int(time.mktime(datetime.datetime.strptime('{:%m/%d/%Y}'.format(date), '%m/%d/%Y').timetuple()) * 1000)
```

Comment:
if you want it a bit simpler, following would work as well:
self['date'] = int(time.mktime(date.timetuple())*1000)